repairnator/repairnator-dockerpool/src/main/java/fr/inria/spirals/repairnator/dockerpool/Launcher.java

```diff
@@ -158,15 +160,29 @@ private void initNotifiers() {
         }
     }
 
-    private List<Integer> readListOfBuildIds() {
-        List<Integer> result = new ArrayList<>();
+    private List<InputBuildId> readListOfBuildIds() {
+        List<InputBuildId> result = new ArrayList<>();
+
         File inputFile = new File(this.config.getInputPath());
 
         try {
             BufferedReader reader = new BufferedReader(new FileReader(inputFile));
             while (reader.ready()) {
                 String line = reader.readLine().trim();
-                result.add(Integer.parseInt(line));
+                String[] buildIds = line.split(",");
```

Comment:
I propose that you put the comma as a constant somewhere in Spoon Core. Particularly if you use it as different places.